tekton/release-pipeline.yaml

```diff
@@ -92,9 +131,18 @@ spec:
       workspace: workarea
       subpath: git
   - name: build
-    runAfter: [precheck]
+    runAfter: [prerelease-precheck]
+    when:
+      - cel: "'$(params.runTests)' == 'true'"
     taskRef:
-      name: golang-build
+      resolver: bundles
+      params:
+        - name: bundle
+          value: gcr.io/tekton-releases/catalog/upstream/golang-build:0.3
```

Comment:
naive query 

I see its still using gcr.io only how it ensure usage of `ghcr.io` :thinking:

Reply:
Yes, catalog tasks are still published as bundles to `gcr.io`. 

The bandwidth consumption associated to `Task` bundles is probably very small compared to that of container images, which is why we started with moving container images. However you are right, it would be good to move `Task` bundles as well eventually.

The [publish pipeline](https://github.com/tektoncd/plumbing/blob/main/tekton/resources/cd/catalog-template.yaml) would probably have to be adapted.

Reply:
Sure thank you @afrittoli for the clarification